src/weatherreport/src/weatherreport.app.src

```diff
@@ -0,0 +1,38 @@
+%% -------------------------------------------------------------------
+%%
+%% riaknostic - automated diagnostic tools for Riak
```

Comment:
riaknostic

Reply:
Its origins are from riaknostic. I would be in favor of just `s/riaknostic/derived from riaknostic/g` everywhere, if that works for you?

Reply:
That works for me.

Reply:
Ok, cool. I've updated all the files to now read `derived from riaknostic - automated diagnostic tools for Riak`.

Reply:
Done.